distversion.patch

```diff
@@ -0,0 +1,16 @@
+Patch provided by Josh Dubois, CrossOver product manager, CodeWeavers.
+--- /dev/null	            	2013-12-07 10:15:02.000000000 -0600
++++ b/include/distversion.h	    2013-12-07 10:14:51.000000000 -0600
+@@ -0,0 +1,12 @@
++/* ---------------------------------------------------------------
++ *   distversion.c
++ *
++ * Copyright 2013, CodeWeavers, Inc.
++ * 
++ * Information from DISTVERSION which needs to find 
++ * its way into the wine tree.
++ * --------------------------------------------------------------- */
++
++#define WINDEBUG_WHAT_HAPPENED_MESSAGE "This can be caused by a problem in the program or a deficiency in Wine. You may want to check <a href=\"http://www.codeweavers.com/compatibility/\">http://www.codeweavers.com/compatibility/</a> for tips about running this application."
++
++#define WINDEBUG_USER_SUGGESTION_MESSAGE "If this problem is not present under Windows and has not been reported yet, you can save the detailed information to a file using the \"Save As\" button, then <a href=\"http://www.codeweavers.com/support/tickets/enter/\">file a bug report</a> and attach that file to the report."
```

Comment:
As the builds we are creating here are nothing official, we might want to avoid having the codeweavers links in here. - I assume they wouldn't be too happy to be bothered with issue reports for something they did not create themselves?

In my version of `distversion.c` I think I changed the wording a bit.

Reply:
According to the note for this patch CodeWeaver _did_ take issues for builds but this was prior to 32on64 so maybe that needs to be altered again. I missed that change and just grabbed my copy of the patch.